.github/workflows/tests.yml

```diff
@@ -20,5 +20,9 @@ jobs:
         run: |
           pip install poetry
           poetry install
+      - name: Format with Black
+        run: |
+          pip install black
```

Comment:
We could also install black using poetry and just run `poetry run black` then we dont need the `pip install`
(same for pytest and pylint when/if we implement them)

Reply:
Yes great catch! lemme check and add it